src/components/Input/Input.styles.ts

```diff
@@ -174,6 +174,21 @@ export const DivStyled = styled.div`
     margin-right: 12px;
 `;
 
+export const BlockieWrapperStyled = styled.div`
```

Comment:
I think the naming convention might not have even existed back then but:
https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md#css-naming-an-element
Could you rename them `DivStyledBlockie` and `DivStyledEllipsis` please?